src/Microsoft.Data.SqlClient/tests/ManualTests/EventSource/EventSourceTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using static Microsoft.Data.SqlClient.SqlClientEventSource;
+
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.EventSourceTest
```

Comment:
1. If not making any connection, consider moving tests to FunctionalTests instead.
2. Fix namespace by referring to other tests.